applications/HDF5Application/python_scripts/core/utils.py

```diff
@@ -12,6 +12,17 @@
 
 import KratosMultiphysics
 
+def EvaluatePattern(pattern: str, model_part: KratosMultiphysics.ModelPart, time_format='') -> str:
+    time = model_part.ProcessInfo[KratosMultiphysics.TIME]
+    pattern = pattern.replace("<time>", format(time, time_format))
+
+    if KratosMultiphysics.STEP in model_part.ProcessInfo:
+        step = model_part.ProcessInfo[KratosMultiphysics.STEP]
+    else:
+        step = 0
```

Comment:
why does `TIME` not get the same treatment?

Reply:
Actually, both does not need this check since `ProcessInfo` is a data value container, and when `ProcessInfo::GetValue` called in serial, if the variable is not found, it adds the zero value of the variable and returns it. So, I removed this check as well.